multiversx_sdk_cli/cli_shared.py

```diff
@@ -147,6 +153,24 @@ def check_broadcast_args(args: Any):
         raise errors.BadUsage("Cannot both 'simulate' and 'send' a transaction")
 
 
+def check_guardian_args(args: Any):
+    if args.guardian_address:
+        if not args.guardian_service_url or not args.guardian_code:
+            raise errors.BadUsage("All guardian arguments must be provided")
+
+    if args.guardian_service_url:
+        if not args.guardian_address or not args.guardian_code:
+            raise errors.BadUsage("All guardian arguments must be provided")
+
+    if args.guardian_code:
+        if not args.guardian_address or not args.guardian_service_url:
+            raise errors.BadUsage("All guardian arguments must be provided")
+
+    if args.guardian_address and args.guardian_service_url and args.guardian_code:
+        if not args.options == 2:
```

Comment:
Here, we should check using a bitwise operator (check the 2nd least significant bit). `options = 3` is also valid (as an example).

Can we perhaps simplify the checks for the error message `All guardian arguments must be provided`? E.g. perhaps using the `all` builtin function? (optional)

Reply:
Now for the options check it's a bitwise operation. 
Also simplified the checks, now using `all()`